app/styles/app/modules/build-messages.scss

```diff
@@ -48,11 +47,25 @@
     }
 
     .message {
+      flex-grow: 1;
       padding-left: 7px;
       border-left: 1px solid $dry-cement;
       margin-left: 2px;
     }
 
+    .message-help svg {
+      display: none;
+      transform: scale(0.8);
+      border-color: $oxide-blue;
+      @include colorSVG($oxide-blue);
```

Comment:
The icon does mot look round:
<img width="954" alt="image" src="https://user-images.githubusercontent.com/4066489/66396857-5128ff00-e9e3-11e9-9cc7-be6340d9e15d.png">

That's because `icon-help` has fixed size (`20px `* `20px`), but width is overwritten to `18px` for `.yml-message-line`. So I'd suggest to overwrite height here as well.

Reply:
done 👍